docusaurus/src/css/custom.css

```diff
@@ -47,6 +50,7 @@ html[data-theme="dark"] {
   --color-grey-40: hsl(240, 14%, 14%);
   --color-grey-400: hsl(240, 13%, 72%);
   --color-green-50: hsl(184, 100%, 12%);
+  --color-dark-blue-30: hsl(240, 56%, 9%);
```

Comment:
I personally feel it looks better to use the `--color-blue-30` from both the light- and dark-theme files from the airbyte platform for this background color - it looks slightly more blue than grey in the light theme, and it looks slightly more grey than blue in the dark theme, which looks nice:
<img width="533" alt="Screenshot 2024-05-16 at 2 20 31 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/0a5c157f-989c-4698-9014-e6334e5f9ee1">
<img width="529" alt="Screenshot 2024-05-16 at 2 20 37 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/ac60f598-60d3-42b7-b74f-29303d0237f1">

What do you think @nora-airbyte and @bnchrch

Reply:
I like it!